templates/python/pydantic_models.py

```diff
@@ -0,0 +1,12 @@
+# You can remove this file.
+# It's only purpose is providing pydantic models for the template.
+from pydantic import BaseModel
+
+
+# You can also just use a typed dict, without Pydantic
+class GreetingRequest(BaseModel):
+    name: str
+
+
+class Greeting(BaseModel):
+    message: str
```

Comment:
I would honestly have them in `example.py`, makes the whole thing way more readable, and this is super small...